### DISCUSSION TITLE
Delta information, e.g., modified

### DISCUSSION THREAD

**Discussion by timkahlke** [ternaustralia/ontology_tern#145] [Q&A] [answered]
I saw that the mapping file  https://github.com/ternaustralia/bdr-faealla-worked-example/tree/5bf4b70f1b60ed47fff7a300a01062b62568e5ad is missing terms like "dcterm:modifiedBy". Is this a final decision or are there plans to model modifications? Or is this part of a larger discussion about deltas.

**Comment by edmondchuc** [comment]
@timkahlke do you mean how you would submit new data that are amendments to existing data already submitted to the BDR?

**Comment by dr-shorthair** [comment, marked as answer]
There is no `dcterms:modifiedBy` in the standard DCMI vocabulary. 
https://www.dublincore.org/specifications/dublin-core/dcmi-terms/

**Reply by timkahlke** [reply]
There is. However that seems to have been ignored/dropped regarding the mapping sheet of the worked example and I was wondering why.

**Reply by dr-shorthair** [reply]
You'll need to show me where it is documented @timkahlke

**Reply by timkahlke** [reply]
[Here](https://docs.google.com/spreadsheets/d/1p3scX7z6wPQ0vtG-Bo_yoYcvRRs8muGm/edit?usp=sharing&ouid=108129827562056706312&rtpof=true&sd=true) is the link to the mapping file from the github repo.

Maybe I'm reading the mapping file wrong. I understand it in the way that everything that's highlighted in green was mapped from the ALA data to the BDR/Tern-Ontology.

**Reply by dr-shorthair** [reply]
Even if it was there, it is not from the DCMI vocabulary. 
It is considered bad form to add definitions to someone else's namespace.

**Reply by edmondchuc** [reply]
@timkahlke the original data has a column `dcterms:modified`. I can't see where it references `dcterms:modifiedBy`.

Assuming you are referring to why we didn't map `dcterms:modified`, there are two reasons:

1. The column exists but there are no values
2. Even if it did contain values, the context of this modified date is about the original data. The only time in the TERN Ontology you would use `dcterms:modified` is on the `RDFDataset` class, and these two instances of `dcterms:modified` are used in different contexts (source data modified date versus RDF data modified date). Mapping the source data's modified date to the RDFDataset's modified date would be misleading.

**Reply by timkahlke** [reply]
Thanks @edmondchuc and yes, you are right, of course, it's `dcterms:modified`. I mis-understood the "NA" in the definitions column as "can't be modelled" instead of "no value".

**Reply by edmondchuc** [reply]
No problem. Feel free to create new discussions to ask new questions. I will mark this thread as answered.
